libnopegl/src/node_colorstats.c

```diff
@@ -131,6 +134,11 @@ static int setup_init_compute(struct colorstats_priv *s, const struct pgcraft_bl
 
     s->depth_index         = ngli_pgcraft_get_uniform_index(s->crafter_init, "depth",         NGLI_PROGRAM_SHADER_COMP);
     s->length_minus1_index = ngli_pgcraft_get_uniform_index(s->crafter_init, "length_minus1", NGLI_PROGRAM_SHADER_COMP);
+
+    char block_name[MAX_ID_LEN];
+    snprintf(block_name, sizeof(block_name), "%s_block", block->name);
+    s->block_init_index = ngli_pgcraft_get_buffer_index(s->crafter_init, block_name, block->stage);
```

Comment:
To prevent this awkward duplication, can't we have a `ngli_pgcraft_get_ublock_index()` (or `main_block`, `uniform_block`, ...) to get a reference to the single auto-inserted block that pgcraft automatically crafted from uniforms?

I also see an amalgam between "block" and "buffer" here, not sure if you want to make it consistent somehow.

Reply:
I can use block here.

Reply:
Changed to get_block_index()